ext/openssl/ossl_asn1.c

```diff
@@ -1818,6 +1842,8 @@ do{\
     rb_define_method(cASN1ObjectId, "oid", ossl_asn1obj_get_oid, 0);
     rb_define_alias(cASN1ObjectId, "short_name", "sn");
     rb_define_alias(cASN1ObjectId, "long_name", "ln");
+    rb_define_method(cASN1ObjectId, "==", ossl_asn1obj_eq, 1);
+    rb_define_alias(cASN1ObjectId, "===", "==");
```

Comment:
Can we remove this alias, I'm not sure it makes sense.

Reply:
I addressed this in 1351ae3630125eb149d03a15bdc6ae8503c359d2